local-volume/provisioner/cmd/main.go

```diff
@@ -70,13 +75,16 @@ func getNode(client *kubernetes.Clientset, name string) *v1.Node {
 	return node
 }
 
-func createDiscoveryMap() map[string]common.MountConfig {
-	// Default setting
-	// TODO: change this to configurable settings.
-	m := make(map[string]common.MountConfig)
-	m["local-storage"] = common.MountConfig{
-		HostDir:  "/mnt/disks",
-		MountDir: "/local-disks",
+func createDiscoveryMap(client *kubernetes.Clientset) map[string]common.MountConfig {
```

Comment:
Can we run through the config map here, and log all the storage class -> directory details that were passed to us?  It could help with debugging issues in the discovery.